dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/ResourcesService.java

```diff
@@ -860,7 +846,7 @@ public Result onlineCreateResource(User loginUser, ResourceType type, String fil
         // save data
         Date now = new Date();
         Resource resource = new Resource(pid,name,fullName,false,desc,name,loginUser.getId(),type,content.getBytes().length,now,now);
-
+        //TODO confirm need to check the success status?
         resourcesMapper.insert(resource);
```

Comment:
@lgcareer hi,Is it necessary to check the success status here?

Reply:
I think not necessary,because here should not be the error status